bitnami/cassandra/values.yaml

```diff
@@ -54,6 +54,24 @@ commonAnnotations: {}
 ##
 commonLabels: {}
 
+## Cassandra container ports to open
+## If hostNetwork true: the hostPort is set identical to the containerPort
+##
+containerPorts:
+  cassandra:
+    intra: 7000
+    tls: 7001
+    jmx: 7199
+    cql: 9042
+    thrift: 9160
+  metrics:
+    http: 8080
+    jmx: 5555
```

Comment:
Please organize container ports as shown below:

```yaml
## Cassandra container ports to open
## If hostNetwork true: the hostPort is set identical to the containerPort
##
containerPorts:
  intra: 7000
  tls: 7001
  jmx: 7199
  cql: 9042
  thrift: 9160

(...)

metrics:

  (...)
  ## Cassandra metrics exporter container ports to open
  ## If hostNetwork true: the hostPort is set identical to the containerPort
  ##
  containerPorts:
    http: 8080
    jmx: 5555
```

Two reasons to do so:
- Consistency with other charts
- The rest of parameters affecting the metrics exporter are also organized under the `metrics` object.